src/main/java/io/federecio/dropwizard/swagger/SwaggerBundle.java

```diff
@@ -60,10 +58,10 @@ public void run(T configuration, Environment environment) throws Exception {
         new AssetsBundle(Constants.SWAGGER_RESOURCES_PATH, configurationHelper.getSwaggerUriPath(), null, Constants.SWAGGER_ASSETS_NAME).run(environment);
 
         environment.jersey().register(new SwaggerResource(configurationHelper.getUrlPattern()));
-        environment.getObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
I believe this is what was causing the issue I was seeing.  Do you get valid swagger specs back the way it currently is?  Thanks for forking this.